packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_single.dart

```diff
@@ -127,14 +127,21 @@ class ClassesTableSingle extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return FlatTable<SingleClassStats>(
-      columns: _columns,
-      data: classes.classes,
-      dataKey: 'ClassesTableSingle',
-      keyFactory: (e) => Key(e.heapClass.fullName),
-      selectionNotifier: selection,
-      defaultSortColumn: _shallowSizeColumn,
-      defaultSortDirection: SortDirection.descending,
+    return ValueListenableBuilder(
```

Comment:
instead of notifying on the filter everywhere, we should have a different value `filteredData` on `classes` that these table's use.  We do not want to be calling `filtered` in a build method as that is potentially expensive business logic.